packages/admin/src/router/PromptHandler.tsx

```diff
@@ -31,13 +46,19 @@ export const RouterPromptHandler: React.FunctionComponent<{}> = ({ children, ...
         return ret;
     };
 
+    const handleClose = async (action: PromptAction) => {
+        const results: Array<AllowTransition> = await Promise.all(promptActions.map(async (promptAction) => await promptAction(action)));
```

Comment:
```suggestion
        const results: Array<AllowTransition> = await Promise.all(promptActions.map((promptAction) => promptAction(action)));
```

There's no need for this additional async and await statements.